<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lista</class>
 <widget class="QDialog" name="Lista">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UŻYTKOWNICY</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/image/icon.png</normaloff>:/image/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Lista
{
	border-image:url(:/image/background.jpg);
}
#label, #label_2, #label_3, #label_4, #label_6, #groupBox
{
	color:white;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>140</y>
     <width>125</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Szukaj</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>525</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>200</y>
     <width>125</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Załaduj tabelę</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>50</y>
     <width>169</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Corbel</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Imie:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Corbel</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nazwisko:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>230</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Corbel</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Urodziny:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_firstname">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>80</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_lastname">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>140</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_birth_date">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>260</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>281</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Corbel</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Zainteresowania:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_interests">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>200</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>25</y>
     <width>900</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Corbel</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Dostępni użytkownicy</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>pushButton</zorder>
  <zorder>tableView</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_4</zorder>
  <zorder>lineEdit_firstname</zorder>
  <zorder>lineEdit_lastname</zorder>
  <zorder>lineEdit_birth_date</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEdit_interests</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEdit_firstname</tabstop>
  <tabstop>lineEdit_lastname</tabstop>
  <tabstop>lineEdit_birth_date</tabstop>
  <tabstop>lineEdit_interests</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
